src/token/CustomERC721.sol

```diff
@@ -23,14 +23,17 @@ import {SalesConfiguration} from "../drops/struct/SalesConfiguration.sol";
 
 import {Address} from "../drops/library/Address.sol";
 import {MerkleProof} from "../drops/library/MerkleProof.sol";
+import {Strings} from "./../drops/library/Strings.sol";
 
 /**
  * @dev This contract subscribes to the following HolographERC721 events:
  *       - customContractURI
  *
  *       Do not enable or subscribe to any other events unless you modified the source code for them.
  */
-contract CustomERC721 is NonReentrant, ERC721H, BatchMintMetadata, DelayedReveal, ICustomERC721 {
+contract CustomERC721 is NonReentrant, ERC721H, LazyMint, DelayedReveal, ICustomERC721 {
+  using Strings for uint256;
```

Comment:
Do we need this library?

Reply:
not anymore, I used it before my last commit

Reply:
okay cool got it